widget/popup_test.go

```diff
@@ -201,3 +201,48 @@ func TestModalPopUp_TappedSecondary(t *testing.T) {
 	assert.Equal(t, 1, len(test.Canvas().Overlays().List()))
 	assert.Equal(t, pop, test.Canvas().Overlays().List()[0])
 }
+
+func TestModalPopUp_Resize(t *testing.T) {
+	label := NewLabel("Hi")
+	win := test.NewWindow(NewLabel("OK"))
+	win.Resize(fyne.NewSize(80, 80))
+	pop := NewModalPopUp(label, win.Canvas())
+	defer test.Canvas().Overlays().Remove(pop)
+
+	assert.Less(t, pop.Content.Size().Width, 70)
+	assert.Less(t, pop.Content.Size().Height, 50)
```

Comment:
If `theme.Padding()` is greater than zero, that's true for the final content size, too.

Reply:
OK. somewhat academic given that the window was 80x80 but I have fixed it anyhow :)

Reply:
My comment has nothing to do with the window size.
I requested that the test should ensure that a size change was made. Setting the size to 70x50 changes the content's size to 70-<pad>x50-<pad>. Thus the resulting size is probably smaller than 70x50. Therefore a test that the previous size was smaller than 70x50 is not an indicator for a content size change.

Reply:
I think I follow - did the updated test fix the issue?

Reply:
Yes